openshift/templates/metabase/metabase.dc.json

```diff
@@ -71,18 +72,47 @@
         "wildcardPolicy": "None"
       }
     },
+    {
+      "apiVersion": "v1",
+      "kind": "Service",
+      "metadata": {
+        "annotations": {
+          "template.openshift.io/expose-uri": "postgres://{.spec.clusterIP}:{.spec.ports[?(.name==\"postgresql\")].port}"
+        },
+        "name": "${NAME_DATABASE}-${SUFFIX}"
+      },
+      "spec": {
+        "ports": [
+          {
+            "name": "postgresql",
+            "port": 5432,
+            "protocol": "TCP",
+            "targetPort": 5432
```

Comment:
might want to make these ports into openshift secrets or something. not sure if they're considered to be sensitive information or not.

Reply:
or maybe as an input param

Reply:
updating to be a param